cla_backend/apps/legalaid/utils/csvupload/constants.py

```diff
@@ -364,3 +145,246 @@
       ( [ABD-HJLNPQ-Z10]{2} )       # unit
     )
 )$"""
+
+
+def get_determination_codes():
+    determination_codes = {u"OOSC", u"OSPF", u"CHNM", u"FINI", u"DVCA"}
+    if settings.CONTRACT_2018_ENABLED:
+        determination_codes |= {"FAFA"}
```

Comment:
@Katy600 and @said-moj 

Here's [Said's earlier work](https://github.com/ministryofjustice/cla_backend/pull/492/files) adding the new FAFA determination code integrated with our new setting that disables the changes until they all ship.